### PR TITLE
Feature/vonk 4187 access policy doc

### PR DESCRIPTION
**Pull request by richfirely**

### CODE REVIEW

File: security/permissions.rst [hunk 1]
```diff
@@ -0,0 +1,103 @@
+
+Permissions (AccessPolicy)
+--------------------------
+Firely Server supports filtering the requested client scopes by using built-in custom access policy resources. Both Smart-V1 and Smart-V2 scopes are supported.
```
Comment (alexzautke) [inline security/permissions.rst:4]: Firely Server supports filtering the **granted** access scopes for a client by using built-in custom access policy resources.

Comment (alexzautke) [inline security/permissions.rst:4]: The access policy decisions are based on HL7 SMART on FHIR scopes, both SMART on FHIR v1 and v2 scopes are supported.

File: security/permissions.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
+
+Permissions (AccessPolicy)
+--------------------------
+Firely Server supports filtering the requested client scopes by using built-in custom access policy resources. Both Smart-V1 and Smart-V2 scopes are supported. 
+The resource definitions are preloaded in Firely Server and can be viewed on the administration endpoint API or in Simplier under
```
Comment (alexzautke) [inline security/permissions.rst:5]: The StructureDefinitions ...

Comment (alexzautke) [reply security/permissions.rst:5]: Typo in "Simplier"

**Review by alexzautke — CHANGES_REQUESTED** [review]

File: security/permissions.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
+
+Permissions (AccessPolicy)
+--------------------------
+Firely Server supports filtering the requested client scopes by using built-in custom access policy resources. Both Smart-V1 and Smart-V2 scopes are supported. 
+The resource definitions are preloaded in Firely Server and can be viewed on the administration endpoint API or in Simplier under 
+`Firely Server Definitions - Access Policy <https://simplifier.net/Vonk-ResourcesR4/~resources?text=access&fhirVersion=R4&sortBy=RankScore_desc>`_ .
```
Comment (louisledev) [inline security/permissions.rst:6]:
```suggestion
`Firely Server Definitions - Access Policy <https://simplifier.net/Vonk-ResourcesR4/~resources?text=accesspolicy&fhirVersion=R4&sortBy=RankScore_desc>`_ .
```

Comment (louisledev) [inline security/permissions.rst:6]: When checking the resource on Simplifier, I see a warning for the AccessPolicyDefinition : 
`The canonical from this resource does not match any claim in this context and conflicts with a claim from another scope.

http://hl7.org/fhir

Canonical claims are used to verify ownership of your canonical URLs.`
When I check the URL field I see indeed:
`"url": "http://hl7.org/fhir/StructureDefinition/AccessPolicyDefinition",`
Also, when checking the "AccessPolicy", I see that the `AccessPolicy.instantiatesCanonical` field refers to a target profile of type: `"http://fire.ly/fhir/StructureDefinition/AccessPolicyDefinition"`

I am not that familiar with FHIR, but it does not seem normal.
Could you confirm?

File: security/permissions.rst [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+The *AccessPolicyDefinition* resource controls the scopes which are permissible. 
+The *AccessPolicy* resource contains the references to Patient, Group, Practitioner, PractitionerRole, Person, RelatedPerson and Device for which the AccessPolicyDefinition applies.
```
Comment (louisledev) [inline security/permissions.rst:8]:
```suggestion
The *AccessPolicy* resource associates Patient, Group, Practitioner, PractitionerRole, Person, RelatedPerson and Device with AccessPolicyDefinition(s).
```

File: security/permissions.rst (hunk 3 above)
Comment (richfirely) [reply security/permissions.rst:6]: @alexzautke. I am not familiar with Simplifier enough to evaluate this. Do you know?

**Review by louisledev — APPROVED** [review]
One minor comment and a question which anyway is not directly concerned by the PR.

File: security/permissions.rst (hunk 2 above)
Comment (louisledev) [inline security/permissions.rst]:
```suggestion
The StructureDefinitions are preloaded in Firely Server and can be viewed on the administration endpoint API or in Simplifier under 
```

Comment (louisledev) [reply security/permissions.rst]: .suo file was added in the last commit. Consider adjusting .gitignore or removing the file

File: build-in-docker.ps1 [hunk 5]
```diff
@@ -1,11 +1,7 @@
...
 Write-Information -Message "Building documentation in folder ${docsroot}" -InformationAction Continue
 docker run --rm -v ${docsroot}:/docs firely.azurecr.io/firely/docs-sphinx:latest
-if ($LASTEXITCODE -ne 0) {
-    Write-Error -Message "Docker run failed. Is Docker active?" -RecommendedAction "Start Docker Desktop" -InformationAction Stop
-    Exit
-}
 $index = "${docsroot}\_build\html\index.html"
 Write-Information -Message "Build ready, showing output from ${index}" -InformationAction Continue
-explorer.exe ${index}
+#explorer.exe ${index}
```
Comment (louisledev) [inline build-in-docker.ps1:6]: Did you mean to commit those changes?

**Pull request merged**